code/modules/projectiles/guns/misc/blastcannon.dm

```diff
@@ -1,26 +1,39 @@
+/**
+ *
+ */
 /obj/item/gun/blastcannon
 	name = "pipe gun"
 	desc = "A pipe welded onto a gun stock, with a mechanical trigger. The pipe has an opening near the top, and there seems to be a spring loaded wheel in the hole."
 	icon_state = "empty_blastcannon"
-	var/icon_state_loaded = "loaded_blastcannon"
 	inhand_icon_state = "blastcannon_empty"
 	w_class = WEIGHT_CLASS_NORMAL
 	force = 10
 	fire_sound = 'sound/weapons/blastcannon.ogg'
 	item_flags = NONE
 	clumsy_check = FALSE
 	randomspread = FALSE
+	/// The icon state used when this is loaded with a bomb.
+	var/icon_state_loaded = "loaded_blastcannon"
 
-	var/hugbox = TRUE
+	/// Whether or not
+	var/hugbox = FALSE
```

Comment:
I doubt anyone will code an hugboxed version of this gun anytime soon. Feel free to remove the variable and its code.

Reply:
Done